geocoder_module/geocoder.py

```diff
@@ -263,6 +264,12 @@ def _get_geocode_info(
 
         return results
 
+    def _check_format(self, location: str) -> bool:
```

Comment:
If the function is returning a true/false, I always like to ask a question in the function name so that it makes it more readable - what do you think? Doesn't have to be my suggestion, just something along those lines?

Also I've removed the `self` property and the underscore at the begging of the function name - as I don't think either are required? feel free to correct me on that though.

```suggestion
    def check_location_can_be_processed(location: str) -> bool:
```

Reply:
Refactored and done